docs/src/componentDocs/MobileStepper/markdown/MobileStepperExamples.mdx

```diff
@@ -1,86 +1,18 @@
-import Box from '@mui/material/Box';
-import MobileStepperDotVariant from '../images/mobileStepper_dot.png';
-import MobileStepperTextVariant from '../images/mobileStepper_text.png';
-import MobileStepperProgressVariant from '../images/mobileStepper_progress.png';
+import {MobileStepperDotVariant} from '../examples/MobileStepperDotVariant'
+import {MobileStepperTextVariant} from '../examples/MobileStepperTextVariant'
+import {MobileStepperProgressVariant} from '../examples/MobileStepperProgressVariant'
 
 ## Mobile Stepper
 
-The basic mobile stepper uses dots to indicate progress.
```

Comment:
You need to keep this description for mobile stepper.

Reply:
Added the description